modules/rosa-hcp-creating-account-wide-sts-roles-and-policies.adoc

```diff
@@ -13,21 +13,21 @@ Before using the {product-title} (ROSA) CLI, `rosa`, to create {hcp-title-first}
 * You have completed the AWS prerequisites for {hcp-title}.
 * You have available AWS service quotas.
 * You have enabled the ROSA service in the AWS Console.
-* You have installed and configured the latest ROSA CLI (`rosa`) on your installation host.
-+
-[NOTE]
-====
-To successfully install {hcp-title} clusters, use the latest version of the ROSA CLI (`rosa`).
-====
+* You have installed and configured ROSA CLI (`rosa`) version 1.2.23 or later on your installation host.
 * You have logged in to your Red Hat account by using the ROSA CLI.
 
 .Procedure
 
-. If they do not exist in your AWS account, create the required account-wide STS roles and policies by running the following command: 
+. If they do not exist in your AWS account, create the required account-wide STS roles and policies by running the following command:
 +
 [source,terminal]
 ----
-$ rosa create account-roles --force-policy-creation
+$ rosa create account-roles --hosted-cp --force-policy-creation
 ----
 +
-The `--force-policy-creation` parameter updates any existing roles and policies that are present. If no roles and policies are present, the command creates these resources instead.
\ No newline at end of file
+The `--force-policy-creation` parameter updates any existing roles and policies that are present. If no roles and policies are present, the command creates these resources instead.
+
+[NOTE]
+====
+ROSA hosted control plane clusters require account and operator roles with AWS managed policies attached. Customer managed policies are not supported. For more information regarding the AWS managed policies for ROSA, see link:https://docs.aws.amazon.com/ROSA/latest/userguide/security-iam-awsmanpol-account-policies.html[AWS managed policies for ROSA] in the AWS documentation.
```

Comment:
We haven't yet enforced managed policies for HCP, but my understanding is that we will enforce managed policies in GA.

Reply:
Please ignore my recent comment, as we are going to enforce managed policies soon in GA, this paragraph is accurate.